files/es/glossary/sql/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SQL
+slug: Glossary/SQL
+tags:
+  - BBDD
+  - Glosario
+  - Sql
+translation_of: Glossary/SQL
+---
```

Comment:
We are adding the last sha reference on content repo https://github.com/mdn/content/blob/main/files/en-us/glossary/sql/index.md

```suggestion
l10n:
  sourceCommit: d2a9f2e26a8139d4bb270d7dc3cddd8b848719fe
---
```

Reply:
Ah, interesting, is this documented somewhere?

Reply:
yes, the main branch is here https://github.com/mdn/mdn-community/discussions/129#discussioncomment-3191138